geometry_msgs/PoseStamped pose
string filename
string name
---